lib/template-mixins.js

```diff
@@ -247,6 +249,7 @@ module.exports = function (fields, options) {
                     return {
                         label: t(label) || '',
                         value: value,
+                        type: opts.type,
                         selected: selected,
```

Comment:
New mixin to add to readme https://github.com/UKHomeOffice/passports-template-mixins/blob/master/README.md#mustache-mixins-available. 

Would be awesome to add it to the HOF example too.